docs/DeveloperGuide.adoc

```diff
@@ -279,104 +298,446 @@ Refer to the guide <<DevOps#, here>>.
+
+|`* * *`
+|student who likes to buy games online
+|use the expense tracker funds left
+|see if I am able to pay my school fees
+
+|`* * *`
+|mobile game addict who makes a lot of in-app purchases
+|use the expenses tracker report
+|manage my finances and to sustain my essential expenses
+
+|`* * *`
+|student that frequently takes on part-time jobs
+|use finance planner to track my income
+|
+
+|`* * *`
+|frugal NUS student
+|use the expense tracker report
+|optimise my finances
+
+|`* * *`
+|social animal frequently going to clubs and bars
+|plan out my expenses
+|moderate my spending on social activities and keep up my balanced lifestyle
+
+|`* * *`
+|student that stays in Hall
+|add my expenses to the expense tracker
+|track funds contributed to hall activities
+
+|`* * *`
+|student who wants to save more money
+|be able to compare my past expenditure with my current expenditure
+|know if I had spent more money or saved money this month
+
+|`* * *`
+|hall resident who eats supper daily
+|keep track of what I have been eating and how much I usually spend on supper
+|don't overspend on each supper
+
+|`* * *`
+|child of wealthy parent
+|easily delete previous expenses
+|properly count my expenditure should my parents decide to help cover some parts of it
+
+|`* * *`
+|careless student
+|make some changes to the information
+|ensure that the application displays the correct information
+
+|`* * *`
+|student who loves to keep track of finance
+|view all the possible commands
+|fully utilize the functionalities offered by the application
+
+|`* *`
+|student that wants to allocate less money to material goods and more to food
+|analyse the spending by category and adjust accordingly
+|
+
+|`* *`
+|on-campus resident who likes to cook my own food
+|track my grocery bills
+|moderate the cost and frequency of grocery shopping
+
+|`* *`
+|motor enthusiast
+|use the planner to set aside funds
+|keep purchasing vehicle parts, modifications and accessories and keep pursuing my hobby
+
+|`* *`
+|student who has a bad habit of spending a lot on material goods
+|see which goods are taking up most of my monthly budget
+|better manage my finances
+
+|`* *`
+|Hypebeast that spends a lot of money on branded goods
+|look at my recent spending percentage
+|manage my finances and keep my budget
+
+|`* *`
+|student who is going for exchange
+|make sure that I bring enough foreign currency
+|avoid using my credit card
+
+|`* *`
+|student who is currently in a relationship
+|the tracker to help me set aside enough money for both my partner and I
+|get married soon
+
+|`*`
+|student who always lends money to people
+|be able to send receipt to my loanee
+|claim back my money
+
+|`*`
+|dota addict who always buy arcanas and battle pass levels
+|track my spending
+|afford to buy more battle pass levels
+
+|`*`
+|active youth in need of sustenance within a limited budget
+|get recommendations for food prices
+|be healthy physically and financially
+
```

Comment:
Might be too many actors in User Stories. Consider narrowing scope of application.